website/docs/compaction.md

```diff
@@ -71,24 +71,25 @@ spark-submit --packages org.apache.hudi:hudi-utilities-bundle_2.11:0.6.0 \
 After compactions have been asynchronously scheduled on the Hudi Timeline, you can now execute those compactions with one of the following options
 
 ### Hudi Compactor Utility
-Hudi provides a standalone tool to execute specific compactions asynchronously. Below is an example and you can read more in the [deployment guide](/docs/deployment#compactions)
-
-:::info
-Note: As of version 0.10.0, the `instant-time` parameter is no longer required for the Hudi Compactor Utility. Each spark-submit will execute the earliest scheduled compaction on the Hudi timeline.
-:::
+Hudi provides a standalone tool to execute specific compactions asynchronously. Below is an example and you can read more in the [deployment guide](/docs/cli#compactions)
```

Comment:
Here we can use `/docs/deployment#compactions` link since `HoodieCompactor` is different from Hudi CLI.

Reply:
Good call, I updated